com.cocoatech.deckr.yaml

```diff
@@ -0,0 +1,128 @@
+app-id: com.cocoatech.deckr
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: deckr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+
+modules:
+  - name: deckr
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      # - mkdir -p Deckr
+      # - tar -xf Deckr-Linux-Portable.tar.gz -C Deckr
+      - cp -r Deckr /app/Deckr
+      - echo $(ls /app/Deckr)
+      - chmod +x /app/Deckr/deckr
+      - mkdir -p /app/bin
+      - ln -s /app/Deckr/deckr /app/bin/deckr
+      - mkdir -p /app/share/metainfo
+      - cp -r com.cocoatech.deckr.metainfo.xml /app/share/metainfo/
+      # - mkdir -p /app/share/icons/hicolor/scalable/apps
+      # - cp -r icon.svg /app/share/icons/hicolor/scalable/apps/com.cocoatech.deckr.svg
+      - mkdir -p /app/share/icons/hicolor/32x32/apps
+      - cp -r app_icon_32.png /app/share/icons/hicolor/32x32/apps/com.cocoatech.deckr.png
+      - mkdir -p /app/share/icons/hicolor/64x64/apps
+      - cp -r app_icon_64.png /app/share/icons/hicolor/64x64/apps/com.cocoatech.deckr.png
+      - mkdir -p /app/share/icons/hicolor/128x128/apps
+      - cp -r app_icon_128.png /app/share/icons/hicolor/128x128/apps/com.cocoatech.deckr.png
+      - mkdir -p /app/share/icons/hicolor/256x256/apps
+      - cp -r app_icon_256.png /app/share/icons/hicolor/256x256/apps/com.cocoatech.deckr.png
+      - mkdir -p /app/share/icons/hicolor/512x512/apps
+      - cp -r app_icon_512.png /app/share/icons/hicolor/512x512/apps/com.cocoatech.deckr.png
+      - mkdir -p /app/share/applications
+      - cp -r com.cocoatech.deckr.desktop /app/share/applications/
+      - mkdir -p /app/share/appdata
+      - cp -r com.cocoatech.deckr.metainfo.xml /app/share/appdata/
+    sources:
+      - type: archive
+        url: https://get.cocoatech.com/dashboard/Deckr-Linux-Portable.tar.gz
+        sha256: 54f4850b28420bc8e2bd67eb4fe19138db5158a6a91163792503ebb21215ff38
+        dest: Deckr
+
+      - type: file
+        path: com.cocoatech.deckr.metainfo.xml
+      - type: file
+        path: com.cocoatech.deckr.desktop
```

Comment:
nor is this file.

Reply:
Are your instruction wrong?

1. Create a new branch with your app’s name: git checkout -b your_app_name
2. Add your app’s manifest to the branch, commit it and push the commit

I did that.

Reply:
![gnome-shell-screenshot-826Z21](https://user-images.githubusercontent.com/287137/230247453-68dfa1da-5a2b-448f-b72f-44158ad83853.png)

Reply:
so, i need to add all these other files to this repo?

I need some instructions.

Reply:
But on https://github.com/flathub/flathub/wiki/App-Requirements#repository-layout

> The manifest must be at the top level and named after the application ID with the extension .json, .yml or .yaml. All patches and data files required also go in the repository.

Also https://github.com/flathub/flathub/wiki/App-Maintenance#acceptable-but-should-be-submitted-upstream

> Additional metadata files (com.example.MyCoolApp.metainfo.xml and com.example.MyCoolApp.desktop) which are mandatory for Flathub apps but not shipped by some (primarily cross-platform) applications

(you acknowledged having read it above)